resources/views/events/export.blade.php

```diff
@@ -186,6 +186,12 @@
 	<tr>
 		<td colspan="2">CONTACTGEGEVENS OUDER</td>
 	</tr>
+
+	<tr>
+		<td>Kamp informatie</td>
```

Comment:
Hoeft wat mij betreft niet in de export. De bedoeling van de export is om 1 document te maken dat de leidingploeg ontvangt op de training, zodat ze weten welke deelnemers ze op kamp hebben en daarop kunnen plannen (speciale eisen/wensen, gevraagde stof, bepaalde allergieën, etc.). De export gaat dan ook doorgaans geprint mee op kamp daarna. Informeren van de ouders loopt altijd via de kantoorcommissie, dus de leiding heeft niets aan deze info.